lib/validators/validator_helpers.rb

```diff
@@ -0,0 +1,13 @@
+module ValidatorHelpers
```

Comment:
Since this module is also used directly in models, would a different name be more appropriate? Simply `DateValidator`?